test/observables.js

```diff
@@ -4,14 +4,14 @@ var expect = require('expect');
 
 var asyncDone = require('../');
 
-var Observable = require('rx').Observable;
+var Observable = require('rxjs').Observable;
 
 function success() {
   return Observable.empty();
 }
 
 function successValue() {
-  return Observable.return(42);
+  return Observable.of(42);
```

Comment:
Can you leave a comment that this was previously `Observable.return` in case someone was using the docs as a reference for older Rx?

Reply:
Or maybe we should keep both dependencies (rx & rxjs) since 4.x is still published as rx??

Reply:
`rx` is only used for the tests so I don't think we need to keep both versions as dependencies but it's definitely valuable to add a comment and mention it in the documentation. I'll do it.